go.mod

```diff
@@ -150,7 +150,7 @@ require (
 	github.com/opencontainers/image-spec v1.1.1 // indirect
 	github.com/opencontainers/runtime-spec v1.2.1 // indirect
 	github.com/openshift/client-go v0.0.0-20251015124057-db0dee36e235 // indirect
-	github.com/openshift/library-go v0.0.0-20251016115829-235f9f1a273e // indirect
+	github.com/openshift/library-go v0.0.0-20251021141706-f489e811f030 // indirect
```

Comment:
My expectation was that the PR contains only the new FBC files. Why do you need this change in this PR?